src/templates/base/2019/index.html

```diff
@@ -4,6 +4,12 @@
 {% block image_height %}562{% endblock %}
 {% block image_width %}715{% endblock %}
 
+{% if featured_chapters_exists(lang, year) or featured_chapters_exists('en', year) %}
+{% set year_with_chapters = true %}
+{% else %}
+{% set year_with_chapters = false %}
+{% endif %}
```

Comment:
Are these the same thing?
```suggestion
{% set year_with_chapters = featured_chapters_exists(lang, year) or featured_chapters_exists('en', year) %}
```

Reply:
Yup this works. Thanks!